### PR TITLE
Clang-format example for FWCore/Framework (v2)

### PR DESCRIPTION
**Pull request by davidlange6**
same as v1 but with
AlwaysBreakTemplateDeclarations: true

### CODE REVIEW

File: .clang-format [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+SortIncludes:    false
+IndentWidth:     2
+AccessModifierOffset: -2
+PenaltyBreakComment: 30
+PenaltyExcessCharacter: 100
+AlignAfterOpenBracket: Align
+AllowShortIfStatementsOnASingleLine: false
+AllowShortLoopsOnASingleLine: false
+BinPackParameters: false
+#AlwaysBreakTemplateDeclarations: false
```
Comment (Dr15Jones) [inline .clang-format]: Does this default to true?

**Comment by Dr15Jones** [comment]
I prefer this format (it is also the format I've always used :) ).

**Comment by makortel** [comment]
I think this format is mostly better.

(ultimately I'd still like to be able to have short declarations/definitions like `template <typename T> class Signal;` on a single line, but if that can't be achieved, let it be)

**Comment by wddgit** [comment]
I also prefer this format. I was just wondering when this might be merged? Not really important to me and certainly if there are issues we should resolve them before merging, but if it is really going to be merged, I would prefer sooner rather than later. I keep having this urge to waste my time fixing the things manually that this fixes for me ...

File: .clang-format (hunk 1 above)
Comment (smuzaffar) [reply .clang-format]: @Dr15Jones , yes default is true/Yes

**Pull request closed without merging**